bundles/org.openhab.binding.freeboxos/src/main/java/org/openhab/binding/freeboxos/internal/handler/ActivePlayerHandler.java

```diff
@@ -73,12 +73,14 @@ protected void internalPoll() throws FreeboxException {
             Player player = getManager(PlayerManager.class).getDevice(getClientId());
             updateStatus(player.reachable() ? ThingStatus.ONLINE : ThingStatus.OFFLINE);
             if (player.reachable()) {
-                Status status = getManager(PlayerManager.class).getPlayerStatus(getClientId());
-                if (status != null) {
-                    updateChannelString(PLAYER_STATUS, PLAYER_STATUS, status.powerState().name());
-                    ForegroundApp foreground = status.foregroundApp();
-                    if (foreground != null) {
-                        updateChannelString(PLAYER_STATUS, PACKAGE, foreground._package());
+                if (isLinked(PLAYER_STATUS + "#" + PLAYER_STATUS) || isLinked(PLAYER_STATUS + "#" + PACKAGE)) {
```

Comment:
Channel IDs and Channel Group IDs are mixed in the constants:
https://github.com/openhab/openhab-addons/blob/73a3f151c94e1fa5956b8b0aa3eb2cc98dadf718/bundles/org.openhab.binding.freeboxos/src/main/java/org/openhab/binding/freeboxos/internal/FreeboxOsBindingConstants.java#L89-L118

So it's error prone using `PLAYER_STATUS` from the Channel ID section as Channel Group ID, although they currently are having the same value.

It was not introduced with this PR, but you could consider creating a dedicated constant prefix by `CHANNEL_GROUP_`:
```java
public static final String CHANNEL_GROUP_PLAYER_STATUS = "player-status";
```

Reply:
Done.
I also renamed other constants for channel group ids.